README

```diff
@@ -0,0 +1,102 @@
+Plugin GIMP : Fourier Transform

```

Comment:
I am not a big fan of duplicating the contents of this file. Is this a problem to use the markdown version?  Or maybe to convert markdown to plain text ? (`pandoc README.md -f markdown -t plain`  may produce a decent text file without markdown tags)

Reply:
I noted that the fedora version copied the README to the doc folder, so I continued the trend. It would probably be simpler to copy README.md to the doc directory.
It is easily possible, but sometimes you need to introduce changes at a slower pace. As you notice, the autoconf/automake is already a very significant change, so it may be sometimes better to delay some other edits until later.
I'm ok with whichever direction you want to go here.
I think I only copied README.md (no README exists) for xinvaders3d, so we are thinking along similar lines here.

Reply:
Ok let's copy README.md into README